src/c/util/time.c

```diff
@@ -1,6 +1,15 @@
 #include <uxr/client/util/time.h>
 #include <uxr/client/config.h>
+
+#if defined(__ZEPHYR__)
+#if ZEPHYR_VERSION_CODE >= ZEPHYR_VERSION(3,1,0)
```

Comment:
```suggestion
#include <version.h>
#if ZEPHYR_VERSION_CODE >= ZEPHYR_VERSION(3,1,0)
```
Add version header for `ZEPHYR_VERSION` macro.